Sources/BraintreePaymentFlow/BTPaymentFlowDriver.m

```diff
@@ -18,35 +19,42 @@
 #import <BraintreePaymentFlow/BTPaymentFlowResult.h>
 #import <BraintreeCore/BTLogger_Internal.h>
 #import <BraintreeCore/BTAPIClient_Internal.h>
+#import <BraintreeCore/BraintreeCore.h>
 
 #endif
 
-@interface BTPaymentFlowDriver () <SFSafariViewControllerDelegate>
+#import <AuthenticationServices/AuthenticationServices.h>
+
+@interface BTPaymentFlowDriver () <ASWebAuthenticationPresentationContextProviding>
 
 @property (nonatomic, copy) void (^paymentFlowCompletionBlock)(BTPaymentFlowResult *, NSError *);
-@property (nonatomic, strong, nullable) SFSafariViewController *safariViewController NS_AVAILABLE_IOS(9_0);
 @property (nonatomic, strong, nullable) id<BTPaymentFlowRequestDelegate> paymentFlowRequestDelegate;
 @property (nonatomic, copy, nonnull) NSString *returnURLScheme;
 @property (nonatomic, strong, nonnull) BTAPIClient *apiClient;
+@property (nonatomic, strong, nonnull) BTPaymentFlowRequest *request;
+@property (nonatomic, strong, nonnull) BTWebAuthenticationSession *webAuthenticationSession;
 
 @end
 
-NSString * const BTPaymentFlowDriverErrorDomain = @"com.braintreepayments.BTPaymentFlowDriverErrorDomain";
+NSString *const BTPaymentFlowDriverErrorDomain = @"com.braintreepayments.BTPaymentFlowDriverErrorDomain";
+NSString *const BTCallbackURLScheme = @"sdk.ios.braintree";
 
 @implementation BTPaymentFlowDriver
 
 static BTPaymentFlowDriver *paymentFlowDriver;
 
-+ (void)load {
-    if (self == [BTPaymentFlowDriver class]) {
-        [[BTAppContextSwitcher sharedInstance] registerAppContextSwitchDriver:self];
-    }
```

Comment:
This was indirectly causing the Venmo UI test to crash. It seemed like it may no longer be needed since ASWebAuthenticationSession handles this internally but I'm wondering if removing this has additional side effects.